spyder/app/tests/test_mainwindow.py

```diff
@@ -1688,7 +1688,8 @@ def test_troubleshooting_menu_item_and_url(monkeypatch):
 @flaky(max_runs=3)
 @pytest.mark.slow
 def test_help_opens_when_show_tutorial_full(main_window, qtbot):
-    """Test fix for #6317 : 'Show tutorial' opens the help plugin if closed."""
+    """Test fix for spyder-ide/spyder#6317 :
+    'Show tutorial' opens the help plugin if closed."""
```

Comment:
This is incorrectly formatted. It needs to be

```python
"""
Test fix for spyder-ide/spyder#6317 :

'Show tutorial' opens the help plugin if closed.
"""
```

Reply:
OK! So never text on a line with """? When there is more than one line?

Btw, should it be a space after the `spyder-ide/spyder#XXXX`? In the code it varies. (I assume it can be a space as here, but is it better with a space?)

Reply:
> OK! So never text on a line with """? When there is more than one line?

If your docstring doesn't fit in a single line (i.e. if it has more than 79 chars), then you need to split it in a first line that contains a summary and subsequent lines (separated by a blank line) that contain a description of the function/method/class being documented.

> Btw, should it be a space after the spyder-ide/spyder#XXXX?

Sorry, I should have suggested you to remove the colon too. But that's not much of an issue.

Reply:
You can remove that colon if you want.